pom.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.solvd.laba</groupId>
+    <artifactId>carina_tests</artifactId>
+    <version>1.0</version>
+    <packaging>jar</packaging>
+    <name>carina_tests</name>
+    <url>https://solvd.com</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <carina-core_version>1.1.3</carina-core_version>
+        <carina-dataprovider.version>1.1.3</carina-dataprovider.version>
+        <carina-api.version>1.1.3</carina-api.version>
+        <carina-aws-s3.version>1.1.4</carina-aws-s3.version>
+        <carina-azure.version>1.1.4</carina-azure.version>
+        <carina-appcenter.version>1.1.4</carina-appcenter.version>
+        <java.version>11</java.version>
+        <zebrunner-agent.version>1.9.6</zebrunner-agent.version>
+        <suite>helloWorld</suite>
```

Comment:
have you tried using Maven to run your tests?

Reply:
Yes, I have run my tests with Maven, and they all passed. I just updated my pom.xml with the correct name. Thank you for pointing it out.